tests/Requests/PostEvent/JobDataTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\NotificationClient\Tests\Requests\PostEvent;
+
+use Keboola\NotificationClient\Requests\PostEvent\JobData;
+use PHPUnit\Framework\TestCase;
+
+class JobDataTest extends TestCase
```

Comment:
Filozoficka.. je potreba delat test serializace samostatne na `JobData`, kdyz je to vlastne soucasti `FailedJobEvent` testu? Jako rikam si, ze jo, protoze to muze kdykoliv v budoucnu mit use-case nezavisly na `FailedJobEvent`, ale jisty si nejsem

Reply:
no filozofická - je to unit test a přidaná hodnota unit testů je místy taková jalová. Ale jak říkáš - spolíhat na to, že teď je to otestovaný nečím jiným můžu teď a v budoucnosti nevím, takže to tam radši nechám.